lib/lazy_high_charts/layout_helper.rb

```diff
@@ -81,8 +81,8 @@ def request_is_referrer?
       defined?(request) && request.respond_to?(:headers) && request.headers["X-XHR-Referer"]
     end
 
-    def request_turbolinks_5_tureferrer?
-      defined?(request) && request.respond_to?(:headers) && request.headers["Turbolinks-Referrer"]
+    def is_turbolinks_5?
+      Gem::Version.new(Turbolinks::VERSION) >= Gem::Version.new('5.0.0')
```

Comment:
Didn't cross my mind. Then i'm at my wit's end with this. I guess there is no way to determine if rails has loaded turbolinks.js with certainty. 
My best bet is that it is only fixable by manually passing an argument that switches the turbolinks wrapper on/off

Anyhow. I guess i can't help on this one then.
Will keep using this for my implementation for the time being though.